lib/game_teams_collection.rb

```diff
@@ -15,6 +15,42 @@ def create_game_teams_array(file_path)
     load_from_csv(file_path, GameTeams)
   end
 
+  def games_by_team_id(team_id)
+    @game_teams_array.select {|game_team| game_team.team_id.to_i == team_id}
```

Comment:
I would change game_teams' team_id attribute to an i so you don't have to .to_i it every time

Reply:
Good catch